src/pages/resource/CreatorSearch.tsx

```diff
@@ -0,0 +1,200 @@
+        setSearchResult(searchResultResponse);
+        setResources(parseStringToJSONAndRemoveParentResource(searchResultResponse, resource.identifier));
+      } catch (error) {
+        setLoadingError(error);
+      } finally {
+        setIsLoading(false);
+      }
+    };
+    searchForCreators();
+  }, [creatorsNames, resource.creators, resource.identifier]);
+
+  const fetchTheRest = async () => {
+    try {
+      //the complete search might already be cached
+      if (searchResult && resources.length < 6) {
+        setIsLoading(true);
+        setLoadingError(undefined);
+        const queryObject: QueryObject = {
+          query: '',
+          offset: 0,
+          limit: searchResult.numFound,
+          institutions: [],
+          resourceTypes: [],
+          licenses: [],
+          tags: [],
+          showInaccessible: false,
+          orderBy: 'created',
+          order: Order.Desc,
+          mine: false,
+          creators: creatorsNames,
+        };
+        const searchResultResponse = (await searchResources(queryObject)).data;
+        setSearchResult(searchResultResponse);
+        setResources(parseStringToJSONAndRemoveParentResource(searchResultResponse, resource.identifier));
+      }
+      setShowEverything(true);
+    } catch (error) {
+      setLoadingError(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <>
+      {(resources.length > 0 || loadingError || isLoading) && (
+        <SearchWrapper>
+          <Typography data-testid="also-published-by-header" gutterBottom variant="h2">
+            {displayCreatorNames.join(', ')}{' '}
+            {displayCreatorNames.length > 1
+              ? t('resource.also_published_by_plural').toLowerCase()
+              : t('resource.also_published_by_singular').toLowerCase()}
+          </Typography>
+          {isLoading ? (
+            <CircularProgress />
+          ) : (
+            <>
+              {loadingError && <ErrorBanner error={loadingError} />}
+              <Grid container spacing={3}>
+                {showEverything ? (
+                  <>
+                    {resources.map((creatorSearchResult) => (
+                      <Grid key={creatorSearchResult.identifier} item>
+                        <CreatorPublishedItem key={creatorSearchResult.identifier} resource={creatorSearchResult} />
+                      </Grid>
+                    ))}
+                  </>
+                ) : (
+                  <>
+                    {resources.slice(0, 5).map((creatorSearchResult) => (
+                      <Grid key={creatorSearchResult.identifier} item>
+                        <CreatorPublishedItem key={creatorSearchResult.identifier} resource={creatorSearchResult} />
+                      </Grid>
+                    ))}
+                  </>
+                )}
+              </Grid>
+              {searchResult?.numFound && searchResult.numFound > 6 && !showEverything && (
```

Comment:
skal vi forresten vise samme liste for flere forfattere eller dele opp 
![image](https://user-images.githubusercontent.com/46489468/124741075-a200bb00-df1b-11eb-943f-9745508a32b2.png)

Reply:
En liste per forfatter? Hvordan løser vi det på flere ressurser de har samarbeidet om?

Reply:
Tja. Hver liste hadde vel inneholdt alle ressursene de har vært med på å publisere selv om det medfører at de samme ressursene kan vises flere ganger ?

Reply:
Fikset